lib/hanami/application.rb

```diff
@@ -147,19 +141,12 @@ def settings
       end
 
       def namespace
-        configuration.namespace
-      end
-
-      def namespace_name
-        namespace.name
+        application_name.namespace
       end
 
+      # TODO: Remove this after https://github.com/hanami/hanami/pull/1156 is merged
       def namespace_path
-        inflector.underscore(namespace)
-      end
```

Comment:
Thanks to https://github.com/hanami/hanami/pull/1156/commits/097b6226f8c98e79cb0a2a1b5b43b50a9ed8ada1 in https://github.com/hanami/hanami/pull/1156, there's no longer the need for this to be public API.